Tasks/Xcode/xcodebuild2.js

```diff
@@ -83,6 +87,10 @@ function processInputs() {
 	xcb.arg('-configuration');
 	xcb.arg(tl.getInput('configuration', true));
 	
+	//Test Results publish inputs
+	publishResults = (tl.getInput('publishJUnitResults', true) == "true");
```

Comment:
The new task lib which is linked into the task has tl.getBoolInput method which does what you're doing here.